syntaxes/csharp.json

```diff
@@ -123,6 +160,43 @@
         }
       ]
     },
+    "genericConstraints": {
+      "begin": "(where)\\s*(\\w+)\\s*:",
```

Comment:
It probably doesn't matter in practice, but shouldn't the `\\s*` after `(where)` above _technically_ be `\\s+`? Otherwise, wouldn't it match `whereImCool:`?